syntrax/src/main/java/org/asciidoctor/diagram/syntrax/Syntrax.java

```diff
@@ -0,0 +1,55 @@
+package org.asciidoctor.diagram.syntrax;
+
+import org.asciidoctor.diagram.*;
+import org.atpfivt.jsyntrax.InputArguments;
+import org.atpfivt.jsyntrax.styles.StyleConfig;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+import static org.atpfivt.jsyntrax.Main.generateSVG;
+import static org.atpfivt.jsyntrax.Main.getStyleConfig;
+
+public class Syntrax implements DiagramGenerator {
+    public static final MimeType DEFAULT_OUTPUT_FORMAT = MimeType.SVG;
+
+    @Override
+    public String getName() {
+        return "syntrax";
+    }
+
+    @Override
+    public ResponseData generate(Request request) throws IOException {
+        MimeType format = request.headers.getValue(HTTPHeader.ACCEPT);
+        if (format == null) {
+            format = DEFAULT_OUTPUT_FORMAT;
+        }
+
+        if (!format.equals(MimeType.SVG)) {
+            throw new IOException("Unsupported output format: " + format);
+        }
+
+        String[] options = new String[0];
+        String optionString = request.headers.getValue(HTTPHeader.OPTIONS);
+        if (optionString != null) {
+            options = optionString.split(" ");
+        }
+
+        InputArguments iArgs;
+        try {
+            iArgs = new InputArguments(options);
+        } catch (Exception e) {
```

Comment:
The correct way to handle: 
```java
catch (ParseException e) {
            throw new IOException(e);
        }
```

Thus we won't loose information about the call stack. And "when parsing input arguments" message does not add any significant information, as this all thing is buried deep in internals of Asciidoctor file processing

Reply:
And of course, we always must catch the most specific exception

Reply:
thanks for the feedback, now fixed!